src/components/GenerateTransfersMenu.tsx

```diff
@@ -73,6 +78,11 @@ export const GenerateTransfersMenu = (props: GenerateTransfersMenuProps): JSX.El
                 Drain safe
```

Comment:
Do we actually have this already?

Reply:
Yes. I added it with the NFT transfers. It even drains all ER721 tokens.